apps/frontend/src/libs/components/button/button.tsx

```diff
@@ -10,7 +12,7 @@ const Button = ({
 	type = "submit",
 }: Properties): React.JSX.Element => {
 	return (
-		<button onClick={onClick} type={type}>
+		<button className={styles["common-button"]} onClick={onClick} type={type}>
```

Comment:
let's shorten it to button for now. We will extend it with other variations later
```suggestion
		<button className={styles["button"]} onClick={onClick} type={type}>
```